tests/test_detect_target.py

```diff
@@ -10,16 +10,18 @@
 
 from modules.detect_target import detect_target
 from modules import image_and_time
+from modules import detections_and_time
 
 
 DEVICE =                        0 if torch.cuda.is_available() else "cpu"
 MODEL_PATH =                    "tests/model_example/yolov8s_ultralytics_pretrained_default.pt"
 OVERRIDE_FULL =                 False  # Tests are able to handle both full and half precision.
```

Comment:
Change this as well: `not torch.cuda.is_available()` . Add a comment that half precision is not supported by CPU.